tests/shipping/Pickup - Pre-filled profile and shipping data - vtexgame1.test.js

```diff
@@ -0,0 +1,3 @@
+import test from './models/Pickup - Pre-filled profile and shipping data'
+
+test('vtexgame1')
```

Comment:
should we test on other accounts too? such as `vtexgame1geo` and `vtexgame1nolean`

Reply:
hmm I think it's worthy! :onit:

Reply:
Just added the test for `vtexgame1geo` and `vtexgame1nolean` as well.